From 1d144322f9b2d33f5ccca89708e3a884ef8a5159 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Fri, 8 Nov 2013 15:51:10 +0800
Subject: [PATCH 0333/1072] ENGR00286971-2 dts: imx6-sabreauto: disabled
 flexcan when max7310 is gone

git://git.freescale.com/imx/linux-2.6-imx.git imx_3.10.17_1.0.0_beta
commit b777b70b3f64546294f30379f20f621527d89162

Flexcan transceiver is using GPIOs from max7310 on i2c3.
Since max7310 is gone on below dts files, so there's no reason
to keep flexcan alive in those dts files.

Note: since flexcan1 is disabled by default, so did not need
to add it.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
Signed-off-by: Hongbo Zhong <hongbo.zhong@windriver.com>
---
 arch/arm/boot/dts/imx6dl-sabreauto-ecspi.dts     |    5 +++++
 arch/arm/boot/dts/imx6dl-sabreauto-gpmi-weim.dts |    5 +++++
 arch/arm/boot/dts/imx6q-sabreauto-ecspi.dts      |    5 +++++
 arch/arm/boot/dts/imx6q-sabreauto-gpmi-weim.dts  |    5 +++++
 4 files changed, 20 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6dl-sabreauto-ecspi.dts b/arch/arm/boot/dts/imx6dl-sabreauto-ecspi.dts
index 919e736..79aa237 100644
--- a/arch/arm/boot/dts/imx6dl-sabreauto-ecspi.dts
+++ b/arch/arm/boot/dts/imx6dl-sabreauto-ecspi.dts
@@ -12,6 +12,11 @@
 	status = "okay";
 };
 
+&flexcan2 {
+	/* max7310_c on i2c3 is gone */
+	status = "disabled";
+};
+
 &i2c3 {
 	/* pin conflict with ecspi1 */
 	status = "disabled";
diff --git a/arch/arm/boot/dts/imx6dl-sabreauto-gpmi-weim.dts b/arch/arm/boot/dts/imx6dl-sabreauto-gpmi-weim.dts
index 06f22e3..229720f 100644
--- a/arch/arm/boot/dts/imx6dl-sabreauto-gpmi-weim.dts
+++ b/arch/arm/boot/dts/imx6dl-sabreauto-gpmi-weim.dts
@@ -13,6 +13,11 @@
 	status = "disabled";
 };
 
+&flexcan2 {
+	/* max7310_c on i2c3 is gone */
+	status = "disabled";
+};
+
 &gpmi {
 	status = "okay";
 };
diff --git a/arch/arm/boot/dts/imx6q-sabreauto-ecspi.dts b/arch/arm/boot/dts/imx6q-sabreauto-ecspi.dts
index a2c1dff..43df09f 100644
--- a/arch/arm/boot/dts/imx6q-sabreauto-ecspi.dts
+++ b/arch/arm/boot/dts/imx6q-sabreauto-ecspi.dts
@@ -12,6 +12,11 @@
 	status = "okay";
 };
 
+&flexcan2 {
+	/* max7310_c on i2c3 is gone */
+	status = "disabled";
+};
+
 &i2c3 {
 	/* pin conflict with ecspi1 */
 	status = "disabled";
diff --git a/arch/arm/boot/dts/imx6q-sabreauto-gpmi-weim.dts b/arch/arm/boot/dts/imx6q-sabreauto-gpmi-weim.dts
index 0d9b74d..392c410 100644
--- a/arch/arm/boot/dts/imx6q-sabreauto-gpmi-weim.dts
+++ b/arch/arm/boot/dts/imx6q-sabreauto-gpmi-weim.dts
@@ -13,6 +13,11 @@
 	status = "disabled";
 };
 
+&flexcan2 {
+	/* max7310_c on i2c3 is gone */
+	status = "disabled";
+};
+
 &gpmi {
 	status = "okay";
 };
-- 
1.7.5.4

